src/ng/q.js

```diff
@@ -376,6 +378,7 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
   }
 
   function processChecks() {
+    // eslint-disable-next-line no-unmodified-loop-condition
```

Comment:
>
> In src/ng/q.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72948617>:
>
> > @@ -376,6 +378,7 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
> >    }
> >
> >    function processChecks() {
> > +    // eslint-disable-next-line no-unmodified-loop-condition
>
> Is this necessary? As far as I understand, it shouldn't.
>
I only added the comments to places where ESLint complained so they're all
necessary. Here the issue is that we're checking queueSize which is not
modified in the loop.